.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## About this PR
+
+## How to review
+
+Scan through the checklist below, take a look at file differences, clean up documentation, style, whitespace.
+In each case below, quickly scan through your code while searching sfor the concerns of that particular line.
```

Comment:
We should move these to the top of the Checklist section or remove them entirely. The How to review section is reserved for the people that opens the PR to help guide reviewers. 

Examples:
https://github.com/Pv3-Labs/gibberle/pull/28
https://github.com/Pv3-Labs/gibberle/pull/4

Reply:
Good note about the "How to Review section" I didn't think of it like that. 

@Exiledz5155 Do you mean adding a PR as a subtask to a ticket? Didn't we say that we'd have PRs on our board? I think it's a good thing to have everything in one place on the board. Also, at any given time there probably wouldn't be many active PRs. So, I don't think it would create clutter.

Imagine our board right now, with 1-3 PRs in a section waiting for review. I feel like that's fine. What do you think?

Reply:
The original ticket will move over to the review columns. I'm ok with adding the PR to the board as a subtask under the original ticket but filtered out on the board itself. This is to prevent a doubling of tickets that are for the same thing.

Example:

Ticket 1 is created to track work for component. If we open a PR and now add it to the board, we are essentially doubling the same ticket.

We can still add the PR to the board, but let's keep it a subtask with the subtask label, linked under the original ticket.

Typically with Jira we wouldn't have this issue as everything is linked properly. Alternatively, instead of creating branches via command line, we exclusively do it through GitHub. This I think would auto link everything tot he original ticket without the need of subtasks